framework/src/st_things/Make.defs

```diff
@@ -52,6 +55,7 @@ include $(THINGS_STACK_DIR)/framework/Make.defs
 include $(THINGS_STACK_DIR)/easy-setup/Make.defs
 include $(THINGS_STACK_DIR)/cloud/Make.defs
 include $(THINGS_STACK_DIR)/security/Make.defs
+include $(THINGS_STACK_DIR)/security/sss_security/Make.defs
```

Comment:
i think you need to check below
`ifeq ($(CONFIG_SVR_DB_SECURESTORAGE),y)`

Reply:
I think so. I will modify it.